src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -760,6 +760,10 @@ def is_windows():
     return platform_name == 'windows'
 
 
+def is_github_codespaces():
+    return os.environ.get('CODESPACES', None) == 'true'
```

Comment:
It's better to put [the document of `CODESPACES`](https://docs.github.com/en/codespaces/developing-in-a-codespace/default-environment-variables-for-your-codespace) here. 

`dict.get` already uses `None` as the `default`: https://docs.python.org/3/library/stdtypes.html#dict.get.

```suggestion
    # https://docs.github.com/en/codespaces/developing-in-a-codespace/default-environment-variables-for-your-codespace
    return os.environ.get('CODESPACES') == 'true'
```